code/game/machinery/computer/communications.dm

```diff
@@ -500,10 +500,13 @@
 	if(ticker.mode.name == "meteor")
 		return
 
-	shuttle_master.cancelEvac(user)
-	log_game("[key_name(user)] has recalled the shuttle.")
-	message_admins("[key_name_admin(user)] has recalled the shuttle - [formatJumpTo(user)].", 1)
-	return
+	if(shuttle_master.cancelEvac(user))
+		log_game("[key_name(user)] has recalled the shuttle.")
+		message_admins("[key_name_admin(user)] has recalled the shuttle - [ADMIN_FLW(user)].", 1)
+	else
+		to_chat(user, "<span class='warning'>The shuttle can not be recalled! Central Orders!</span>")

```

Comment:
Maybe re-word this. "Central Orders!" seems strange. Perhaps just delete that bit. Or change it to "Central Command refuses to recall the shuttle."